### PR TITLE
Defaulting ocs-ci to ODF 4.10

### PR DESCRIPTION
**Pull request by vavuthu**
Signed-off-by: vavuthu <vavuthu@redhat.com>

### CODE REVIEW

**Review by jilju — APPROVED** [review]

**Comment by petr-balogh** [comment]
@vavuthu let's wait with this till we will also change stable pipeline to 4.10 and will have agreement on when we will switch.
Usually we do like week or two after release - so let's not merge it day after release yet please. Added WIP to subject to prevent merge for now

**Review by clacroix12 — APPROVED** [review]

**Review by petr-balogh — APPROVED** [review]

**Pull request merged**
